src/client/state.rs

```diff
@@ -474,7 +474,8 @@ mod tests {
                            assert_eq!(broker.map(|b| b.1), state.find_broker(topic, id));
                            (id, broker)
                        })
-                       .collect::<Vec<_>>()[..]);
+                       .collect::<Vec<_>>()
+                        [..]);
     }
```

Comment:
Hmm, this is a little strange. It does still do this for me, though. Maybe you have an older rustfmt? Anyway, it is a little weird, but harmless to style, I think.